DOC: Update contributing guidelines (#539)
skypyproject/skypy: CONTRIBUTING.rst

Contributor Guidelines
======================

How to contribute
-----------------
We love contributions! SkyPy is open source,
built on open source, and we'd love to have you hang out in our community.
Whether you would like to contribute to SkyPy with your own piece of code or

> helping SkyPy develop a concrete feature of the SkyPy roadmap:

Reviewer: Is 'SkyPy' roadmap a specific document? It reads here as if it is, which the reader does not have access to.

You could just replace 'SkyPy roadmap' with 'SkyPy' throughout.

Author: > Is 'SkyPy' roadmap a specific document? It reads here as if it is, which the reader does not have access to.
> 
> You could just replace 'SkyPy roadmap' with 'SkyPy' throughout.

agreed

1. Read through our `Discussions Page`_ to start a new conversation and share your
ideas or follow up an existing conversation on a feature of the roadmap.

> 2. Upon agreement, open an issue describing the feature you will contribute with and 

Reviewer: ```suggestion
2. Following the discussions, when you have a good idea of the specifics of the feature you wish to contribute, open an `Issue`_ describing the feature.
```

> follow the guidelines in this page.

Reviewer: ```suggestion
Then follow the guidelines on the rest of this page to open a Pull Request to contribute the code implementing the new feature.
```

 .. _Discussions Page: https://github.com/skypyproject/skypy/discussions

>  

Reviewer: ```suggestion
 .. _Issue: https://github.com/skypyproject/skypy/issues
```
